### PR TITLE
Update setup to use compose zip and release binaries.

### PR DESCRIPTION
**Pull request by jeffmendoza**
Also update all demos to just point to setup and not rehash the cloning and building of binaries, etc.

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -39,53 +39,48 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
...
+1. Download the GUAC CLI `guacone` binary for your machine's OS and architecture
+   from the
+   [latest GUAC release](https://github.com/guacsec/guac/releases/latest). For
+   example Linux x86_64 is
+   [`guacone-linux-amd64'](https://github.com/guacsec/guac/releases/latest/download/guacone-linux-amd64).
 
-   ```bash
-   git clone https://github.com/guacsec/guac.git
-   ```
+1. Rename the binary to `guacone`, mark it executable if necessary, and add it
```
Comment (pxp928) [inline setup.md:52]: not sure if we need to give this much detail but should we add how to mark it executable and add it to shell's path for inexperienced users?

File: setup.md [hunk 2]
```diff
@@ -39,53 +39,48 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
...
+   to your shell's path.
 
-2. Optional: If you want test data to use, clone GUAC’s test data:
+1. Download the compose files from the
+   [latest GUAC release](https://github.com/guacsec/guac/releases/latest). Look
+   for an attached file `guac-compose-<release-tag>.tar.gz`. At the time of
+   writing, this is:
+   [`guac-compose-v0.1.1.tar.gz`](https://github.com/guacsec/guac/releases/download/v0.1.1/guac-compose-v0.1.1.tar.gz).
```
Comment (pxp928) [inline setup.md:59]: I wonder if we remove this as we will have to update as new releases come

File: setup.md [hunk 3]
```diff
@@ -39,53 +39,48 @@ GUAC components work together]({{ site.baseurl }}{%link guac-components.md %}).
...
-From your GUAC directory, run:
-
-```bash
-make container
-```
-
-## Step 3: Start the GUAC server
+## Step 2: Start the GUAC server
 
 1. In another terminal, from your GUAC directory, run:
```
Comment (pxp928) [inline setup.md:74]: Should this not be the same terminal window and within `guac-compose` directory?

File: querying-via-cli.md [hunk 4]
```diff
@@ -77,12 +49,12 @@ be easily queried.
 Once ingested you will see the following message:
 
 ```bash
-{"level":"info","ts":1681821120.162612,"caller":"cmd/files.go:181","msg":"[2.158961542s] completed doc {Collector:FileCollector Source:file:///../guac-data/docs/spdx/spdx_vuln.json}"}
+{"level":"info","ts":1681821120.162612,"caller":"cmd/files.go:181","msg":"[2.158961542s] completed doc {Collector:FileCollector Source:file:///guac-data-main/docs/spdx/spdx_vuln.json}"}
```
Comment (pxp928) [inline querying-via-cli.md:52]: good change here for consistency

**Review by pxp928 — APPROVED** [review]
LGTM, couple of comments

File: setup.md (hunk 1 above)
Comment (jeffmendoza) [reply setup.md:52]: To do it thoroughly, we would need a matrix of instructions for OS and even Linux distribution, (ex: do you have ~/.profile or ~/.bash_profile ...). There is probably somewhere to link out to where someone has done this well. I'll do some searching.

About the separate doc. All the demos just need this setup.md done with compose up and guacone in the path. Users shouldn't need to download all the binaries if they don't need to for just the demos. I think a "run guac from scratch" with a tutorial on what each binary does would be a good doc to have, but have it be a separate thing, not a prereq.

File: setup.md (hunk 3 above)
Comment (jeffmendoza) [reply setup.md]: I think other terminal is fine, updated directory.

**Pull request merged**